lib/rubocop/cop/style/raise_args.rb

```diff
@@ -80,7 +80,8 @@ def correction_exploded_to_compact(node)
           return node.source if message_nodes.size > 1
 
           argument = message_nodes.first.source
-          exception_class = exception_node.const_name || exception_node.receiver.source
+          exception_class = exception_node.const_name || \
```

Comment:
Any particular reason for the use of `\` here?

Reply:
That's a simply mistake. I'll fix it.